### ISSUE TITLE
How to read a user input?

### ISSUE THREAD

**Issue by Kray-G** [Kray-G/kinx#150]
From https://github.com/Kray-G/gitex/issues/1

**Comment by Kray-G** [comment]
If you want to get an input until pressing [Enter] key, use `$stdin.readLine()`.
This function will care about UTF8.

```
var a = $stdin.readLine();
System.println(a);
```

If you want to get a character without waiting for pressing [Enter] key, use `$stdin.geti()` with the following code.
I am sorry that now I do not prepare the function to get an input easily.

```
var a;
while (true) {
    a = $stdin.geti();
    if (a > 0) {
        var l = System.isUtf8Bytes(a);
        if (l > 1) {
            var s = [a];
            while (--l) {
                s.push($stdin.geti());
            }
            a = s;
        }
        a = *a;
        break;
    }
}
System.println(a);
```

I will prepare the function to wrap the above code in the next release.

**Comment by Kray-G** [comment]
I will introduce a `getUtf8Char()` method to File object including `$stdin` object for the above purpose.
The above example will be rewritten below.

```
var a = $stdin.getUtf8Char();
```

**Comment by Kray-G** [comment]
Included in the release v0.20.1
